### ISSUE TITLE
Major Problems with Berryboot?

### ISSUE THREAD

**Issue by AumCoin** [maxnet/berryboot#397]
Last night I spent a bunch of time trying to get this working on my RPi3.  After an hour I tried a brand new SD card and it worked.  "Great", I thought.  Then the next day I wanted to wipe the thing out and start over with the same SD card.  Now this SD card will not boot with Berryboot, just like the one I replaced last night.  I'm kind of afraid to try any more SD cards at this point.  Any advice?

**Comment by AumCoin** [comment]
I want to add that, with both SD cards that did not work with Berryboot (written from different card read/writers on different ports, host machines, and with different cards), I still have no trouble directly burning any image of any OS made for the RPi3.  I always thought it was neat that I didn't have to actually burn an image to use Berryboot, but the novelty has seriously worn off for me.  I'm going to just make a simple suggestion here, and that is that you include a .img file as an alternative to the "unzip to the root of the SD card" method.  It's the way every other Raspbbery Pi OS is installed and works without fail 100% of the time.  The method you are using apparently does not work all of the time.  I'm not here to debate the "possibility" of your method not working, but I will tell you that it does not work, at least with the latest release, on a working Raspberry Pi 3 Model B.

**Comment by AumCoin** [comment]
I closed this issue because it is resolved, but I have seen so many people report this issue that I would like to describe how I solved it.  Apparently, because I had used the card previously for a Raspberry Pi OS (Raspbian or something), that OS had partitioned the card.  When I put the card in my windows machine it popped up with a drive letter, but that drive letter corresponded to the tiny "BOOT" partition used by Raspbian, while the others were not visible to Windows.

The problem was resolved as follows (from command line):

C> DISKPART
  > LIST DISK     (note the disk #)
  > SELECT DISK 1   (or whatever disk # you noted previously)
  > CLEAN

Then go into Disk Manager and create your partition (or "Simple Volume"), using the whole space.  Then copy the files to the root of the SD card.  Now it works.
